ckan/views/user.py

```diff
@@ -28,6 +28,12 @@
 user = Blueprint(u'user', __name__, url_prefix=u'/user')
 
 
+@user.errorhandler(403)
+def error_handler(e):
+    extra_vars = {u'code': e.code, u'content': e.description}
```

Comment:
There's some extra logic we need to copy from the error controller, specially the one that bypasses templates rendering for the api and fanstatic, otherwise we'll get http errors on the api:

https://github.com/ckan/ckan/blob/master/ckan/controllers/error.py#L35

The rest I don't think we need for now